model/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -25,6 +26,7 @@ case class SubmissionRequest(
   entityType: Option[String],
   entityName: Option[String],
   expression: Option[String],
+  retryType: RetryStatus,
```

Comment:
I don't think we want to modify SubmissionRequest, since it's used in other cases where we don't have retry arguments. We'll need a separate request class